### PR TITLE
feat: ability to configure log driver and log options

### PR DESCRIPTION
**Pull request by sombriks**
*Issue #254*

*Description of changes:*

For my workflow i need to override the `awslogs-group` value so it doesn't clash with other group names. app on dev environment and app on staging would override messages if the task definition file is shared between them, and turns out it is.

By having a chance to override log configuration i can rename the property without having to keep different versions of the same task definiton.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: index.js
Comment (KollaAdithya) [inline index.js:78]: Currently for the ECS task, these are the log configurations that are supported.

For tasks on AWS Fargate, the supported log drivers are `awslogs, splunk, and awsfirelens`
For tasks hosted on Amazon EC2 instances, the supported log drivers are `awslogs, fluentd, gelf, json-file, journald, logentries, syslog, splunk, and awsfirelens`.

Are we assuming these drivers for ECS Tasks based on Ec2 instance? if this is the case do we need to add `logentries` as well ?

File: index.js
Comment (KollaAdithya) [inline index.js:79]: can this be simplified?
```
if (!validDrivers.includes(logConfigurationLogDriver)){
        throw new Error(`'${logConfigurationLogDriver}' is invalid logConfigurationLogDriver. valid options are ${validDrivers}. More details: https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_LogConfiguration.html`)
      }
```

**Review by KollaAdithya — COMMENTED** [review]
Thanks for the PR ❤️ ! I just left some comments feel free to address those comments before we can merge the PR.

File: index.js
Comment (sombriks) [reply index.js:78]: my current scenario is for fargate but yes, let's add logentries!

File: index.js
Comment (sombriks) [reply index.js:79]: i could remove the link to the documentation or the valid options list. what do you think?

Comment (KollaAdithya) [reply index.js:79]: what i mean by this [comment ](https://github.com/aws-actions/amazon-ecs-render-task-definition/pull/255/files#r1281174921)is that

instead of using array function `if (!validDrivers.find(driver => driver === logConfigurationLogDriver))`
can you just use `if (!validDrivers.includes(logConfigurationLogDriver))`?

Comment (sombriks) [reply index.js:79]: ahhh right away!

**Review by KollaAdithya — APPROVED** [review]

**Comment by KollaAdithya** [comment]
Looks like i do not have any write access to merge the PR. I will contact the right person to merge this PR.

**Comment by jonathanhuang13** [comment]
Yeah, can we create a release? Looks like a release hasn't been created since January

**Pull request merged**
